src/Nethermind/Nethermind.Blockchain.Test/Validators/TxValidatorTests.cs

```diff
@@ -19,13 +19,23 @@
 using Nethermind.Serialization.Rlp;
 using Nethermind.Specs;
 using Nethermind.Specs.Forks;
+using Nethermind.Specs.Test;
 using NSubstitute;
 using NUnit.Framework;
 
 namespace Nethermind.Blockchain.Test.Validators;
 
 public class TxValidatorTests
 {
+    private const int MaxBlobCount = 6;
+    private const int TargetBlobCount = 3;
+
+    private IReleaseSpec CancunSpec = new OverridableReleaseSpec(Cancun.Instance)
+    {
+        MaxBlobCount = MaxBlobCount,
+        TargetBlobCount = TargetBlobCount,
+    };
+
```

Comment:
why do we need to do this? just set these default values in Cancun release spec?